CHANGES.md

```diff
@@ -1,5 +1,27 @@
 ## unreleased
 
+## v2.5.0
+
+### New features
+* add context reinitialisation (PR #490)
+* add Dolmen frontend (PR #491,#541,#545)
+* add model generation (PR #530)
+* support mutually recursive definitions in the native language (PR #549, #550)
+* support of some options of the SMT-LIB statement (set-option) (PR #608)
+* support for the (get-model) statement (required the Dolmen frontend) (PR #614)
+* support ground reasoning with some bitvector primitives (PR #730, #733, #745).
+* improve the ite reasoning (simplification of some ites) (PR #731)
+
+### Build
+* update to the new version of ocplib-simplex (0.5)
+* remove the support of the deprecated library num. Alt-Ergo only uses Zarith (PR #600)
+* remove the deprecated graphical interface (PR #601)
+
+### Bug fixes
+* issue 475 -- Catch the exception I_dont_known in instantiation pass of SatML (PR #478)
+* fix a leak memory in the vector module (PR #606)
+* issue 737 (PR #738)
```

Comment:
Also could be described?

Reply:
I didn't describe it because it's not an issue about a released version but an issue about the model generation on the branch next.

Reply:
I removed it. The issue is minor.

Reply:
OK, I think removing it is right then -- if it only affects unreleased versions it probably shouldn't go into the changelog